Framework/API/inc/MantidAPI/IFunction1D.h

```diff
@@ -80,6 +80,14 @@ class MANTID_API_DLL IFunction1D : public virtual IFunction {
                                const size_t nData);
 
 protected:
+  /// Calculate histogram data for the given bin boundaries.
+  virtual void histogram1D(double *out, double left, const double *right,
+                           const size_t nBins) const;
+  /// Devivatives of the histogram.
```

Comment:
It took me a while to understand that this is the derivative of the integral by the different parameters of the fit function. Maybe explain this more clearly here in the doxygen?